metricbeat/module/windows/service/service_windows.go

```diff
@@ -60,12 +75,34 @@ var serviceStates = map[ServiceState]string{
 	ServiceStopped:         "Stopped",
 }
 
-var serviceStartTypes = map[ServiceStartType]string{
-	ServiceAutoStart:   "Automatic",
-	ServiceBootStart:   "Boot",
-	ServiceDemandStart: "Manual",
-	ServiceDisabled:    "Disabled",
-	ServiceSystemStart: "System",
+type serviceStartType int
+
+const (
+	Boot = iota
```

Comment:
I think you want to specify your `serviceStartType` type here. Without it these are `int`s.

`Boot serviceStartType = iota`

Also consider prefixing the constants with `StartType` or similar.

Reply:
These constants are exported from the package (capital letter), but `serviceStartType` is not. In general they should be same.

If the constants are used by any exported functions (like as a param) or as an exported field in an exported type then I would make this type exported. Otherwise keep it private.

I like to run `GOOS=windows godoc -http=:6060 &` then browse to http://localhost:6060/pkg/ and look at the documentation for my functions and types to see if I can make sense of how to use the package. If you have inadvertently made some types private that need to be public you'll often notice the discrepancy when reading the docs. Conversely if there are functions/types in the documentation that have no public use case then make them unexported.